Automattic/jetpack
# Writng Prompts: bloganuary label update

Adds "bloganuary" label to prompt block in the editor if the prompt is from January
implements request discussed here p1704469480742409-slack-C03NLNTPZ2T.

Also add the "bloganuary" icon, and update the link to existing answers to use the bloganuary tag.

Note: There is caching involved in the call to `'/wpcom/v3/blogging-prompts'` API method from https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/jetpack/extensions/blocks/blogging-prompt/edit.js#L97 . I can't work out how to clear the cache that it uses, it may be easiest to open in a new incognito window.

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
 Sync code to a wpcom sandbox to test changes to public API.
Load the editor for a January prompt e.g.
`https://wordpress.com/post/$site_slug?answer_prompt=1813`
You should see the bloganuary icon and "Bloganuary writing prompt" as the label for the block.
The "View all responses" should take you to the bloganuary tag page e.g. bloganuary-2024-10
Save the post, reload, you should still see all of the bloganuary customizations.
Bloganuary!
